src/ActionList/List.tsx

```diff
@@ -167,8 +179,10 @@ export function List(props: ListProps): JSX.Element {
     groups = [...groupMap.values()]
   }
 
+  const {containerRef} = useFocusZone({bindKeys: FocusKeys.ArrowVertical | FocusKeys.HomeAndEnd})
+
   return (
-    <StyledList {...props}>
+    <StyledList ref={containerRef as React.RefObject<HTMLDivElement>} {...props}>
```

Comment:
Can we avoid the `as` here?

Reply:
I had trouble making typescript happy here without it - though this pattern seems to be used other places in the code for this specific use case of the focusZone/focusTrap hooks.

I think there is going to be some changes incoming to how focusZones/focusTraps are implemented into ActionMenu/DropdownMenu/ActionList - I will keep an eye on how that work settles out. This might be something good to pair on that we can fix in a few places if we can find a better pattern.